### PR TITLE
[WD-6717] order partner logos according to copy doc in /cloud/public-cloud

### PR DESCRIPTION
**Pull request by akbarkz**
## Done

- Order partner logos in /cloud/public-cloud according to the [copy doc](https://docs.google.com/document/d/1ZzUIh9Tfz5SvkrHyo4-izzmUjRka1Tn0UQisKcXGEPA/edit?disco=AAAA7Ee_wf8)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /cloud/public-cloud, scroll down to third section and check the order of the logos of the partners and make sure they are listed in the same order as in the copy doc.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6717

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: webapp/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from webapp.app import app
+
+
+# this is template filter that will sort a provided list of dicts
+# by a key in order of values that are provided in `ordered_list`
+@app.template_filter()
+def sort_by_key_and_ordered_list(list_to_sort, obj_key, ordered_list):
```
Comment (albertkol) [inline webapp/__init__.py:7]: I would not put this in the `__init__.py` nothing wrong with it, but we use `handlers.py` to register filters (ex: Line 181) or other app contexts, then we define the function inside `context.py`. I think it's nicer to keep things consistent.

**Review by albertkol — COMMENTED** [review]
Comment.

File: webapp/__init__.py (hunk 1 above)
Comment (akbarkz) [reply webapp/__init__.py]: okay, I was searching for `@app.template_filter()` and couldn't find it anywhere, now I see how it's done. Thanks, Albert! Fixed.

**Review by albertkol — APPROVED** [review]
:+1: Masterfully mastered!

**Pull request merged**